packages/ionic/test/text-control.spec.ts

```diff
@@ -66,3 +75,62 @@ describe('Text control Base Tests', textBaseTest(testConfig, 'ion-input', toSele
 describe('Text control Input Event Tests',
          textInputEventTest(testConfig, 'ion-input', toSelect));
 describe('Text control Error Tests', textErrorTest(testConfig, errorTest));
+describe('Text control type tests', textTypeTest(testConfig, 'ion-input', toSelect));
+
+describe(
+    'Ionic text control', () => {
+        let fixture: ComponentFixture<any>;
+
+        beforeEach(() => {
+            TestBed.configureTestingModule({
+                declarations: [StringControlRenderer],
+                imports: [IonicModule.forRoot(StringControlRenderer)],
+                providers: [
+                    {provide: Platform, useClass: PlatformMock},
+                    {provide: NgRedux, useFactory: MockNgRedux.getInstance}
+                ]
+            }).compileComponents();
+
+            MockNgRedux.reset();
+            fixture = TestBed.createComponent(StringControlRenderer);
+        });
+
+        it('should display error', () => {
+            const schema = {
+                type: 'object',
+                properties: {
+                    foo: {
+                        type: 'string',
+                        format: 'email'
+                    }
+                }
+            };
+            const mockSubStore: Subject<any> = setupMockStore(fixture, {
+                ...defaultTextTestData,
+                schema
+            });
+            mockSubStore.next({
+                jsonforms: {
+                    core: {
+                        data: defaultTextTestData.data,
+                        schema,
+                        errors: [{
+                            dataPath: 'foo',
+                            message: `should match format "email"`
```

Comment:
why do we need to test for erros here, isn't error checking part of the error tests (`textErrorTest(testConfig, errorTest)`)?

Reply:
Fixed